oideslurm/static/slurm.js

```diff
@@ -1,14 +1,40 @@
 'use strict';
 
-angular.module('oide.slurm', ['ngRoute','ui.bootstrap','schemaForm','ui.ace','smart-table'])
-
-.config(['$routeProvider','schemaFormProvider', 'schemaFormDecoratorsProvider', 'sfPathProvider',
- function($routeProvider,schemaFormProvider,  schemaFormDecoratorsProvider, sfPathProvider) {
-  $routeProvider.when('/slurm', {
-    templateUrl: '/static/slurm/slurm.html',
-    controller: 'SbatchCtrl'
-  });
+angular.module('oide.slurm', ['ngRoute','ui.bootstrap','schemaForm','ui.ace','smart-table', 'ui.router'])
+
+.config(['$stateProvider','$urlRouterProvider', 'schemaFormProvider', 'schemaFormDecoratorsProvider', 'sfPathProvider',
+ function($stateProvider, $urlRouterProvider, schemaFormProvider,  schemaFormDecoratorsProvider, sfPathProvider) {
+  // $routeProvider.when('/slurm', {
+  //   templateUrl: '/static/slurm/slurm.html',
+  //   controller: 'SbatchCtrl'
+  // });
   //Add to the bootstrap directive
+    $stateProvider.state('slurm', {
+      'url': '/slurm',
+      'views': {
+        '': {
+          templateUrl: '/static/slurm/slurm.html'
+        },
+        'schedule@slurm': {
+          templateUrl: '/static/slurm/templates/schedule.html'
+        },
+        'jobstatus@slurm': {
+          templateUrl: '/static/slurm/templates/job_status.html'
+        },
+        'directives@slurm': {
```

Comment:
Is it possible to nest directives, sbatch, and sbatchscript views within the schedule view?